### DISCUSSION TITLE
Not showing next days in calendar

### DISCUSSION THREAD

**Discussion by oguzcatal** [iSoron/uhabits#1846] [Help & FAQ]
Hello, I started to use the app recently but there is a thing that I don't understand of find in the settings. On 7th of October I created a habit. For example I want to do somenthing once in two days. Calendar goes backwords to the past. But I wanna see it like 7-8-9-10-11 so when time comes I check that day. Is this possible or am I misssing somethhing here?

**Comment by Quit4944** [comment]
yes, I would also like an option to show the days after "today", so I can check the boxes in the following days, because I have some habits that I anticipated today but which are valid for tomorrow or for the day after tomorrow or for 5 days after
